packages/react-input/src/stories/InputBlock.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Label } from '@fluentui/react-label';
+import { useId } from '@fluentui/react-utilities';
+import { Input } from '../index';
+
+export const Block = () => {
+  const id = useId('input');
+
+  return (
+    <div>
+      <Label style={{ display: 'block', paddingBottom: '2px' }} htmlFor={id}>
+        Block input
+      </Label>
+      <Input style={{ display: 'flex' }} id={id} />
```

Comment:
The thing that makes me think it might be worth calling out is because just using `display: block` will break the internal layout. But I'm open to other ways of demonstrating achieving the layout.

Reply:
If we keep it I'd prefer to write it like this as, in general, there should be no need to change the element's display property for layout purposes.

```tsx
<div style={{ display: 'flex', flexDirection: 'column' }}>
    <Label htmlFor={id}>Stacked Input</Label>
    <Input id={id}/>
</div>
```